addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -1632,6 +1626,13 @@ $-product-radius-map: (
             }
         }
     }
+
+    // Indicator Positioning
+    .o_wsale_product_page_opt_image_width_100_pc &:not(.o_carousel_product_left_indicators) {
+        .o_carousel_product_indicators {
+            @include o-position-absolute(auto, 0, .75rem, 0);
+        }
+    }
```

Comment:
It introduced a little side effect, if the `o_carousel_product_indicators` are at the bottom they are now on the image, before they were under it. It seems to be due to the missing media query.
<img width="483" height="456" alt="image" src="https://github.com/user-attachments/assets/8f1b1b37-bfdd-4244-9b7d-95a12dbca6ab" />